libsolidity/ast/Types.cpp

```diff
@@ -2507,6 +2507,9 @@ string FunctionType::externalSignature() const
 {
 	solAssert(m_declaration != nullptr, "External signature of function needs declaration");
 
+	if (m_declaration->name().empty())
```

Comment:
I use it to determine that it is a fallback. Should we just use `name().empty()` or rather introduce a helper `isFallback()`?